DependencyInjection/Configuration.php

```diff
@@ -136,6 +136,10 @@ private function getDbalConnectionsNode()
                     ->defaultValue(false)
                     ->info('Enables collecting backtraces when profiling is enabled')
                 ->end()
+                ->booleanNode('profiling_collect_schema_errors')
```

Comment:
There is [an ongoing discussion to rename the "profiler" as "inspector"](https://github.com/symfony/symfony/issues/34042), so I think we should avoid that vocabulary, but given there are already `profiling`, and `profiling_collect_backtrace` options, I'm torn…

Reply:
We shouldn't rename it until symfony does. And it won't happen until at least Symfony 6.